samples/javascript_nodejs/51.cafe-bot/dialogs/dispatcher/mainDispatcher.js

```diff
@@ -85,32 +99,44 @@ module.exports = {
          *
          * This method examines the incoming turn property to determine
          * 1. If the requested operation is permissible - e.g. if user is in middle of a dialog,
-         *     then an out of order reply should not be allowed.
+         *    then an out of order reply should not be allowed.
```

Comment:
> then an out of order reply should not be allowed. [](start = 13, length = 50)

I'm trying to square this statement with interruptions.  Is that what this is suggesting?

Reply:
That's right.